conf/web-raw

```diff
@@ -8,7 +8,7 @@ Include /etc/_APP_NAME_/addons/apache2/includes/server/*.conf
 
   Include /etc/_APP_NAME_/addons/apache2/includes/vhost/*.conf
 
-  ProxyPass / http://127.0.0.1:_APP_PORT_/ retry=0
-  ProxyPassReverse / http://127.0.0.1:_APP_PORT_/
+  ProxyPass _APP_SUBFOLDER_OR_ROOT_ http://127.0.0.1:_APP_PORT__APP_SUBFOLDER_/ retry=0
```

Comment:
We do still need some hacking here since 

`http://127.0.0.1:_APP_PORT__SERVER_PATH_PREFIX_/`

may contain a double trailing slash, and as far as my testing goes, that (single) slash is required in order for the proxy to work.